### DISCUSSION TITLE
is docs not the same as the actual code?

### DISCUSSION THREAD

**Discussion by EssaAlshammri** [lorinkoz/django-pgschemas#290] [Q&A] [answered]
in the docs it says:

``` python
MIDDLEWARE = (
    "django_pgschemas.middleware.TenantMiddleware",
    # ...
)
```

but the actual file is in route

am I missing something?

**Comment by lorinkoz** [comment, marked as answer]
Apologies for the confusion. I'm preparing a major v1 release: https://github.com/lorinkoz/django-pgschemas/discussions/277

Please install the latest stable version, which should match the docs, or give a try to the v1 alpha, whose docs are here: https://django-pgschemas.readthedocs.io/en/latest/
